docs/development/translations.rst

```diff
@@ -0,0 +1,300 @@
+
+|Weblate project page screenshot|
+
+- :ref:`Start translating a language on Weblate <how_to_translate_a_language_weblate>`
+- :ref:`How to translate a phrase on Weblate <how_to_translate_a_phrase_weblate>`
+- :ref:`Learn more about translating phrases with placeholders <translating_phrases_with_placeholders>`
+- :ref:`Learn more about translating phrases with HTML code <translating_phrases_with_html>`
+
+SecureDrop translations are a collaborative endeavour!
+
+- :ref:`Suggesting changes to source strings <suggesting_changes_to_source_strings>`
+- :ref:`Changing an existing translation <changing_an_existing_translation>`
+
+.. getting_help_translations:
+
+Getting help about translations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Should you need help, you can do one of the following:
+
+* Post a message in the `translation category of the SecureDrop forum`_
+* Chat in the `SecureDrop instant messaging channel`_
+* Read the `Weblate documentation`_
+
+How-to Guides
+-------------
+
+.. _how_to_register_on_weblate_email:
+
+How to register an account on Weblate using an email address
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Visit the `Weblate registration page`_.
+#. Fill the form **Register using email** and click **Register**.
+#. Check your email for a message from **weblate@securedrop.org** with the subject **[Weblate] Your registration on Weblate**.
+#. That message contains a confirmation link. Click that link to complete your registration.
+
+.. _how_to_register_on_weblate_github:
+
+How to register an account on Weblate using a GitHub account
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Visit the `Weblate registration page`_.
+#. Click on the GitHub icon, under **Third party registration**.
+#. Log into GitHub unless you already are.
+#. Click the green **Authorize freedomofpress** button.
+
+The authorization request looks like this:
+
+|GitHub authorization page screenshot|
+
+.. _how_to_manage_your_preferred_languages_weblate:
+
+How to manage your preferred languages on Weblate
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Visit the `Weblate dashboard`_.
+#. Click the **Manage your languages** button.
+#. Select the languages your want to translate.
+#. Click the **Save** button.
+
+|Weblate manage languages screenshot|
+
+.. _how_to_translate_a_language_weblate:
+
+How to translate a language on Weblate
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Visit the `Weblate dashboard`_.
+#. Click on the **component** in order to display the list of languages in which it is translated.
+#. Click the **Translate** button.
+#. :ref:`Start translating <how_to_translate_a_phrase_weblate>`.
+
+|Weblate translations screenshot|
+
+.. _how_to_translate_a_phrase_weblate:
+
+How to translate a phrase on Weblate
```

Comment:
Maybe cross-reference background section on HTML content in this section?

Reply:
Yes! Good idea. I think the cross-referencing could/should be quite dense to allow for non-linear reading. (Specially when starting from the **How-to Guides**.)

Reply:
Done.